python/lsst/ts/m2gui/controltab/tab_actuator_control.py

```diff
@@ -33,7 +57,516 @@ class TabActuatorControl(TabDefault):
+
+        return layout
+
+    def _create_group_script_control(self):
+        """Create the group of script control. The cell control system reads
+        the binary script to perform a seris of actuator motions.
+
+        Returns
+        -------
+        group : `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        layout = QVBoxLayout()
+
+        layout_file = QFormLayout()
+        layout_file.addRow("Script File:", self._info_script["file"])
+        layout_file.addRow("Progress:", self._info_script["progress"])
+
+        layout.addLayout(layout_file)
+
+        layout_control = QGridLayout()
+        layout_control.addWidget(self._buttons_script["load_script"], 0, 0)
+        layout_control.addWidget(self._buttons_script["stop"], 0, 1)
+        layout_control.addWidget(self._buttons_script["resume"], 0, 2)
+
+        layout_control.addWidget(self._buttons_script["run"], 1, 0)
+        layout_control.addWidget(self._buttons_script["pause"], 1, 1)
+        layout_control.addWidget(self._buttons_script["clear"], 1, 2)
+
+        layout.addLayout(layout_control)
+
+        return create_group_box("Script Control", layout)
+
+    def _create_group_force_summary(self):
+        """Create the group of force summary. This is just for the high-level
+        overview of force status.
+
+        Returns
+        -------
+        group : `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        layout = QFormLayout()
+
+        layout.addRow("Minimum Axial (N):", self._labels_force["axial_min"])
+        layout.addRow("Maximum Axial (N):", self._labels_force["axial_max"])
+        layout.addRow("Total Axial (N):", self._labels_force["axial_total"])
+
+        self.add_empty_row_to_form_layout(layout)
+
+        layout.addRow("Minimum Tangent (N):", self._labels_force["tangent_min"])
+        layout.addRow("Maximum Tangent (N):", self._labels_force["tangent_max"])
+        layout.addRow("Total Tangent (N):", self._labels_force["tangent_total"])
+
+        return create_group_box("Measured Force Summary", layout)
+
+    def _create_group_actuator_selector(self, spacing=1, num_column=10):
+        """Create the group of actuator selector to select single or multiple
+        actuators to do the movements.
+
+        Parameters
+        ----------
+        spacing : `int`, optional
+            Spacing between the items on grid layout. (the default is 1)
+        num_column : `int`, optional
+            Number of column in the grid layout of actuator selection. (the
+            default is 10)
+
+        Returns
+        -------
+        group : `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        layout = QVBoxLayout()
+
+        # Grid layout of actuator selection
+        layout_actuator_selection = QGridLayout()
+        layout_actuator_selection.setSpacing(spacing)
+
+        num_row = int(np.ceil(NUM_ACTUATOR / num_column))
+        idx = 0
+        for row in range(num_row):
```

Comment:
It might be better to organize this by members of layout_actuator_selection and calculate row and column - see ButtonGrid. Also using a generic method. similar to ButtonBox, will benefit the code.